src/init/starship.ps1

```diff
@@ -1,153 +1,157 @@
 #!/usr/bin/env pwsh
 
-function Get-Arguments {
-    function Get-Cwd {
-        $cwd = Get-Location
-        $provider_prefix = "$($cwd.Provider.ModuleName)\$($cwd.Provider.Name)::"
-        return @{
-            # Resolve the actual/physical path
-            # NOTE: ProviderPath is only a physical filesystem path for the "FileSystem" provider
-            # E.g. `Dev:\` -> `C:\Users\Joe Bloggs\Dev\`
-            Path = $cwd.ProviderPath;
-            # Resolve the provider-logical path 
-            # NOTE: Attempt to trim any "provider prefix" from the path string.
-            # E.g. `Microsoft.PowerShell.Core\FileSystem::Dev:\` -> `Dev:\`
-            LogicalPath =
-                if ($cwd.Path.StartsWith($provider_prefix)) {
-                    $cwd.Path.Substring($provider_prefix.Length)
-                } else {
-                    $cwd.Path
-                };
+$null = New-Module starship {
+    function Get-Arguments {
```

Comment:
Please inline `Get-Arguments` again (`--continuation` doesn't need it after all, and it makes the whole thing a bit more readable).